pkg/operator/secretannotator/vsphere/reconciler.go

```diff
@@ -129,6 +144,17 @@ func (r *ReconcileCloudCredSecret) Reconcile(request reconcile.Request) (reconci
 }
 
 func (r *ReconcileCloudCredSecret) validateCloudCredsSecret(secret *corev1.Secret) error {
+	// We only support passthrough so make sure the operator mode is either unset or "passthrough"
```

Comment:
Thought it worth mentioning that the installer will not allow the mode to be set to "passthrough". Of course, the user can set the mode directly in the config CR afterwards.